includes/kaltura_derivatives.inc

```diff
@@ -68,14 +67,38 @@ function islandora_video_streaming_kaltura_upload($object, $force = false) {
 
       $entity = kaltura_entry_create(array('kaltura_entryid' => $entry->id));
       kaltura_map_properties($entry, $entity);
-
-      kaltura_save_entry_metadata($entity);
+      $entity->kaltura_title = $object->id; 
+      $entity->kaltura_description = $object->label;
+      
       kaltura_entry_save($entity); //write entity to drupal db
-
+    
+      try {
+        $kaltura_baseEntry = new KalturaBaseEntry();
+        $kaltura_baseEntry->name = $object->id;
+        $kaltura_baseEntry->description = $object->label;
+        $updated_kaltura = $client->baseEntry->update($entry->id, $kaltura_baseEntry);
+      } catch (Exception $e) {
+        $messages = array(
+          'message' => t('Failed to edit metadata in Kaltura'),
```

Comment:
The parameter `@pid` in `message_sub` isn't used in `message`.

Reply:
Please add the `@pid` into the message.  Without context of what object had the error, the error message will be of limited value in the watchdog.